eq-author-api/schema/resolvers/base.js

```diff
@@ -206,7 +206,11 @@ const Resolvers = {
       const questionnareComments = await getCommentsForQuestionnaire(
         questionnaireId
       );
-      return questionnareComments.comments[id] || [];
+      let sortedComments = questionnareComments.comments[id] || [];
+      sortedComments = sortedComments.sort(
+        (a, b) => b.createdTime - a.createdTime
+      );
+      return sortedComments;
```

Comment:
Alternatively, you can do it all in one expression:

```suggestion
      return comments[id]
        ? comments[id].sort((a, b) => b.createdTime - a.createdTime)
        : [];
```

(assuming you accept the destructuring suggestion)